docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+      # kafka_id: "KAFKA_ID"
+     openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+----
+
+.. Delete the service account using the `delete_service_account_by_id` module.
++
+.Example `delete_service_account_by_id` module
+[source,yaml]
+----
+- name: Delete Service Account
+   rhoas.rhoas.delete_service_account_by_id:
+   # service_account_id: "service_account_id"
+  service_account_id: "{{ srvce_acc_resp_obj['client_id'] }}"
+
+  # openshift_offline_token: "OFFLINE_TOKEN"
+----
+
+.. Deprovision and delete the {product-kafka} instance using the `delete_kafka_by_id` module.
++
+.Example `delete_kafka_by_id` module
+[source]
+----
+- name: Delete kafka instance by ID
+    rhoas.rhoas.delete_kafka_by_id:
+     kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+     openshift_offline_token: "offline_token"
+----
+
+As the playbook deletes all created resources, some alteration of the playbook is necessary to keep them. For more information on writing playbooks, see the https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#playbooks-tips-and-tricks[Ansible documentation].
+
+[id="con-terraform_{context}"]
+== Terraform
+
+link:https://www.terraform.io/[HashiCorp Terraform^] is an infrastructure as code tool that lets you build, change, and version infrastructure safely and efficiently through human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle.
+
+The link:https://registry.terraform.io/providers/redhat-developer/rhoas/latest[RHOAS Terraform^] provider is available in the official link:https://www.terraform.io/[Terraform provider registry^] and includes resources to interact with {product-long-rhoas}.
+
+You can fully manage your Kafka environment through your Terraform system using the RHOAS Terraform provider. You can create, retrieve info, configure, deploy, and delete the following {product-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+
+[id="proc-using-rhoas-terraform-provider_{context}"]
+=== Installing the RHOAS Terraform provider
+
+.Prerequisites
+* You have a Red Hat account.
+* https://www.terraform.io/downloads[Terraform^] v1.3.4 or later is installed.
+* You have an offline token that is accessible to Terraform.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and can be found at https://cloud.redhat.com/openshift/token. As the offline token is a sensitive value that varies between environments it is best specified through as `_OFFLINE_TOKEN_` environment variable when running `terraform apply` in a terminal.
+
+.Procedure
+. Open your browser of choice and navigate to the RHOAS Terraform provider.
```

Comment:
"In your web browser, open the RHOAS Terraform provider at `https://registry.terraform.io/providers/redhat-developer/rhoas/latest`"